src/table/utils/styling-utils.ts

```diff
@@ -34,15 +33,15 @@ export const SELECTION_STYLING = {
 /**
  * Determines if a palette color is set or not. Both index !== -1 and color !== null must be true for it to be unset
  */
-export const isColorSet = (prop: PaletteColor | undefined): boolean =>
+export const isColorSet = (prop?: PaletteColor): boolean =>
```

Comment:
Not a fan of this. This makes `isColorSet()` a valid call, which it is not. There is no point of calling this function if you are not passing an argument

Reply:
okay, gonna revert it.

Reply:
Done.
But when `export const isColorSet = (prop: PaletteColor | undefined): boolean =>`,  isColorSet() is also a valid call.